### PR TITLE
An attempt to use the windows-machine-config-bootstrapper-tests image for e2e windows testing

### PR DESCRIPTION
**Pull request by JacobTanenbaum**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml [hunk 1]
```diff
@@ -55,3 +55,8 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: windows-e2e-ovn
+  commands: ""
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml]: What forced this to be added? You shouldn't have to

File: ci-operator/step-registry/windows/e2e/test/windows-e2e-test-ref.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ref:
+  as: windows-e2e-test
+  from: windows-machine-config-bootstrapper-tests
+  commands: windows-e2e-test-commands.sh
+  resources:
+   requests:
+      cpu: 10m
+      memory: 100Mi
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/windows/e2e/test/windows-e2e-test-ref.yaml]: Does your test really use so little?

File: ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml (hunk 1 above)
Comment (JacobTanenbaum) [reply ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml]: it is a holdover from my ovn-e2e PR which probably got it from a doc or another workflow that is out of date. 

I removed it

File: ci-operator/step-registry/windows/e2e/test/windows-e2e-test-ref.yaml (hunk 2 above)
Comment (JacobTanenbaum) [reply ci-operator/step-registry/windows/e2e/test/windows-e2e-test-ref.yaml]: increased the requests, really just kept the numbers from the configure step of e2e, which is not the same as running the test. Increased the requests

**Comment by JacobTanenbaum** [comment]
@aravindhp @ravisantoshgudimetla  this is a POC and how I think this should work. This PR successfully calls out to your image and tries to execute the windows test. you guys can feel free to use it.

**Comment by aravindhp** [comment]
@PratikMahajan take over this PR once https://github.com/openshift/windows-machine-config-bootstrapper/pull/172 merges to test out your changes from the SDN repo.

**Pull request closed without merging**